charts/rancher-cis-benchmark/5.7.0/templates/validate-install-crd.yaml

```diff
@@ -0,0 +1,17 @@
+#{{- if gt (len (lookup "rbac.authorization.k8s.io/v1" "ClusterRole" "" "")) 0 -}}
```

Comment:
Should this file not be also exist in https://github.com/rancher/cis-operator/tree/release/v1.1/chart/templates?

Reply:
i have added it in the first commit now..but running make charts deletes this file automatically from the packages folder.